bin/prover_checkers/batch_availability_checker

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
```

Comment:
Please add:
```
  set -o errexit   
  set -o pipefail
```
to make the script fail if something went wrong.

The same in other bash scripts.